### ISSUE TITLE
[Azure Toolkit][A11yMAS][Accessibility][Keyboard][MAS2.1.1][IntelliJ][Wipro]: "Refresh" button is not accessible with keyboard in "select subscription" window.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1435]
**Repro Steps:**

1. Open and Start IntelliJ IDEA.
2. Open your project with IntelliJ IDEA
3. Navigate to Tools menu->"Azure"-> "Azure Sign In".
4. In the Azure Sign In window, select "Interactive" radio button, and navigate to "Sign In" button and select it.
5. In the Azure Login Dialog window, enter your "Azure credentials", and then select "Sign in" button.
6. In the Select Subscriptions dialog , navigate to "Refresh" button and try to access using keyboard.

**Expected Result:**
"Refresh" button should  accessible with keyboard in subscription window.

**Actual Result:**
"Refresh" button is not accessible with keyboard in subscription window.

**Platform:**
IntelliJ IDEA 2017.3.2 (Community edition)
Build# IC-173.4127.27
JRE:1.8.0_152-release-1024-b8amd64
JVM: OpenJDK 64 Bit Server VM by Jetbrains.
**OSBuild Version:** Version 1709(Os Build 16299.192)

![refresh button is not accessible with keyboard in subscription](https://user-images.githubusercontent.com/22956130/35726442-c4d16b80-07b9-11e8-8329-2d4762b225b7.png)

**Comment by jdneo** [comment]
Reference: https://youtrack.jetbrains.com/issue/IDEA-191322

**Comment by Eskibear** [comment]
External issues tracked in #1627

**Comment by Eskibear** [comment]
* **Importance of experience**: minimal. Once the dialog is open, it automatically refresh the subscriptions, so the feature is hardly ever used.
* **Quality of experience**: not impacted. A simple workaround: the user can re-open the dialog, it refreshes the list automatically. 
* **Alternate solutions**: No alternate technologies or options can be leveraged.
* **Evidence**: All evidence is provided in this issue and external issue.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749905
